packages/twenty-server/src/engine/metadata-modules/field-metadata/dtos/default-value.input.ts

```diff
@@ -31,7 +31,7 @@ export class FieldMetadataDefaultValueString {
 
 export class FieldMetadataDefaultValueRawJson {
   @ValidateIf((_object, value) => value !== null)
-  @IsObject()
+  @IsObject() // TODO: Should this also allow arrays?
```

Comment:
@FelixMalfait What do you think of this?

Reply:
Let's keep it like that imo